l5kit/l5kit/rasterization/box_rasterizer.py

```diff
@@ -137,11 +141,11 @@ def rasterize(
                     im = draw_boxes(self.raster_size, world_to_image_space, np.append(agents, av_agent), 255)
                     im_ego = draw_boxes(self.raster_size, world_to_image_space, agent_ego, 255)
 
-            agents_im.append(im)
-            ego_im.append(im_ego)
+            agents_im[..., i] = im
+            ego_im[..., i] = im_ego
```

Comment:
Can we do assignment to these in the loop and get rid of those temp variables? I think it would be cleaner to understand.

Reply:
I'll give the variable better names, but replacing with direct assignement is too error prone imo